tests/Behat/MinkContext.php

```diff
@@ -85,6 +103,50 @@ public function assertActiveLanguageSwitcherLink($label): void {
     $this->assertSession()->elementTextContains('css', $selector, $label);
   }
 
+  /**
+   * Asserts that the breadcrumb contains a certain number of elements.
+   *
+   * @param string $trail
+   *   Comma separated breadcrumb trail.
+   *
+   * @throws \Behat\Mink\Exception\ElementNotFoundException
+   * @throws \Behat\Mink\Exception\ExpectationException
+   *
+   * @Then the breadcrumb trail should be :items
+   */
+  public function checkBreadcrumbTrail(string $trail): void {
+    $trail_elements = explode(',', $trail);
+    $selector = 'ol.ecl-breadcrumb__segments-wrapper li.ecl-breadcrumb__segment a';
+    $breadcrumb = $this->getBreadcrumb();
+    $this->assertSession()->elementsCount('css', $selector, count($trail_elements), $breadcrumb);
+    $breadcrumb_elements = $breadcrumb->findAll('css', $selector);
+
+    foreach ($trail_elements as $key => $trail_element) {
```

Comment:
We can make here an assertion that will help showing the difference between the expected and actual breadcrumb trail.
If you do something like this (see tests/Kernel/ContentLanguageSwitcherTest.php:140):
```
$actual = array_map('trim', array_column(iterator_to_array($breadcrumb_elements), 'nodeValue'));
Assert::assertEquals($trail_elements, $actual);
```